Playbooks/Comment-MCASAlertURL

```diff
@@ -0,0 +1,303 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "PlaybookName": {
+            "defaultValue": "Comment-MCASAlertURL",
+            "type": "string"
+        },
+        "UserName": {
+            "defaultValue": "<username>@<domain>",
+            "type": "string"
+        }
+    },
+    "variables": {
+        "AzureSentinelConnectionName": "[concat('azuresentinel-', parameters('PlaybookName'))]",
+        "MCASConnectionName": "[concat('mcas-', parameters('PlaybookName'))]",
```

Comment:
The MCASConnectionName should be removed. the attached file has this removed. MCAS is not a required connection for the Logic App to run successfully. To receive value from the Logic App, MCAS should be a data connection for Sentinel.